webapp/src/clj/lipas/backend/legacy/api.clj

```diff
@@ -0,0 +1,17 @@
+(ns lipas.backend.legacy.api
+  (:require [lipas.data.types-old :as types-old]
```

Comment:
Better use `lipas.data.types` namespace. `types-old` and `types-new` are temporary namespaces to facilitate an ongoing larger change in Lipas types.

Reply:
Changed to `lipas.data.types`.